src/components/input/Button.js

```diff
@@ -42,7 +42,7 @@ const ButtonNext = function Button({
     <ButtonBase
       {...htmlAttributes}
       classes={classnames(
-        // NB: Base classes are applied by ButtonBase
+        'focus-visible-ring transition-colors whitespace-nowrap flex items-center',
```

Comment:
If this common set of styles is not part of the base component, it could potentially be shared via a package-internal constant as well.